internal/storage/activities/aip_deletion_report_test.go

```diff
@@ -79,13 +80,38 @@ func expectLocation(t *testing.T, msvc *fake.MockService) {
 		Return(loc, nil)
 }
 
-func defaultExpects(t *testing.T, msvc *fake.MockService, aipID uuid.UUID) {
+func expectUpdateDeletionRequest(msvc *fake.MockService, drDBID int) {
+	msvc.EXPECT().
+		UpdateDeletionRequest(
+			mockutil.Context(),
+			drDBID,
+			mockutil.Func(
+				"Updates deletion report key",
+				func(updater persistence.DeletionRequestUpdater) error {
+					_, err := updater(&types.DeletionRequest{})
```

Comment:
Just realized while working on another tests, we could check the actual values being updated:

http://github.com/artefactual-sdps/enduro/blob/main/internal/workflow/local_activities_test.go#L173-L181